### DISCUSSION TITLE
Play all songs from an artist

### DISCUSSION THREAD

**Discussion by woonaval** [navidrome/navidrome#2543] [Q&A]
Hello,
Maybe it's possible to do it and I haven't seen the way, but I would like, from the web interface, to click on an Artist, and have the option to enqueue all of their songs, or play them in Random mode. The same as we can do already from an Album view, but for the Artist view. Would that be possible?
Thank you!

**Comment by CyanBC** [comment]
To add to this Sonixd is able to do this without issue.
https://github.com/jeffvli/sonixd
It would be great if the web Navidrome had a similar option.
